core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -28,20 +28,25 @@ oppia.directive('answerGroupEditor', [
         getOnSaveAnswerGroupRulesFn: '&onSaveAnswerGroupRules',
         getOnSaveAnswerGroupCorrectnessLabelFn: (
           '&onSaveAnswerGroupCorrectnessLabel'),
+        onSaveContentIdsToAudioTranslations: '=',
         outcome: '=',
+        correctnessFeedbackEnabled: '=',
```

Comment:
This can be &, right? We won't be editing it within the directive...

I.e. isCorrectnessFeedbackEnabled: '&correctnessFeedbackEnabled'

Reply:
Done, moved this to EditorStateService.